src/middlewared/middlewared/etc_files/local/nslcd.conf

```diff
@@ -0,0 +1,42 @@
+<%
+        def safe_call(*args):
+            try:
+                val = middleware.call_sync(*args)
+            except:
+                val = False
+            return val
+
+        ldap = safe_call('datastore.query', 'directoryservice.LDAP')
+        if ldap and ldap[0]:
+            ldap = ldap[0]
+            capath = None
+            if ldap['ldap_certificate']:
+                cert = safe_call('certificateauthority.query', [('id', '=', ldap['ldap_certificate']['id'])], {'get': True})
+                if cert:
+                    capath = cert['cert_certificate_path']
```

Comment:
I have no idea what I'm talking about but in the next file there is an update:
```                if cert:
-                    capath = cert['cert_certificate_path']
+                    capath = cert['certificate_path']```
So one of them must be wrong, I guess.

Reply:
So apparently cert_certificate_path is the correct key. This was changed at some point, which is why you see a bunch of cert_certificate_path to certificate_path changes. Now that I see that cert_certificate_path is the correct key,  I need to revisit all of these PR's where that key change is made. Good find.